messages/md.retrieve.json

```diff
@@ -0,0 +1,48 @@
+{
+  "retrieveCmd": {
+    "description": "retrieve metadata from an org using Metadata API\nUses Metadata API to retrieve a .zip of XML files that represent metadata from the targeted org. The default target username is the admin user for the default scratch org. You can retrieve and deploy up to 10,000 files or 400 MB (39 MB compressed) at one time.",
+    "examples": [
+      "$ sfdx force:mdapi:beta:retrieve -r path/to/retrieve/dir",
+      "$ sfdx force:mdapi:beta:retrieve -r path/to/retrieve/dir -k package.xml",
+      "$ sfdx force:mdapi:beta:retrieve -r path/to/retrieve/dir --unzip --zipfilename apexClasses.zip"
+    ]
+  },
+  "reportCmd": {
+    "description": "check the status of a metadata retrieval\nSpecify the job ID and a target directory for the retrieve you want to check. You can also specify a wait time (minutes) to check for updates to the retrieve status. If the retrieve was successful, the resulting zip file will be saved to the location passed in with the retrieve target parameter.",
+    "examples": [
```

Comment:
Add example descriptions before each example (please fix if I got it wrong):

"Check the status of the most recent deploy:",
"Wait for 15 minutes, and if the request is complete, retrieve the files into the specified directory:",
"Check status, and if the request is complete, give the zip a custom name and automatically unzip it:"